eng/common/scripts/Helpers/PSModule-Helpers.ps1

```diff
@@ -11,7 +10,8 @@ function Update-PSModulePathForCI()
   if ($IsWindows) {
     $hostedAgentModulePath = $env:SystemDrive + "\Modules"
     $moduleSeperator = ";"
-  } else {
+  }
+  else {
```

Comment:
Just a nit, what do you like about this half indent format?

Reply:
This is my `Powershell` vscode extension being opinionated. I didn't even notice the code change.

Reply:
Huh, that's bizarre, as I think the actual powershell convention is no newline :D